sal_interfaces/CBP/CBP_Telemetry.xml

```diff
@@ -7,42 +7,42 @@
         <item>
             <EFDB_Name>panic</EFDB_Name>
             <Description>The status of the CBP.</Description>
-            <IDL_Type>float</IDL_Type>
+            <IDL_Type>boolean</IDL_Type>
```

Comment:
Please consider making `status` an event. it's all state-like information that would be more useful if reported only when it changes.

I realize this is a pre-existing issue, but I really hope you will consider changing it on this ticket branch.

Reply:
Please update the descriptions to say what the fields mean. For example, for this field I am guessing that panic= "True" means the CBP in in panic state, but the description suggests otherwise and I don't really know what a panic state is (though it sounds bad).

The others are even less clear. "azimuth=True" is good? bad? What is it telling me? The description offers no clue.

I realize this is a pre-existing issue, but I beg you to fix it on this ticket branch.